action.yml

```diff
@@ -13,7 +13,15 @@ inputs:
   command:
     description: 'Command to run.'
     required: false
-    default: 'cargo-kani --workspace'
+    default: 'cargo-kani'
+  working-directory:
```

Comment:
I don't think this need to be an input... isn't `working-directory` a github thingy already?

Reply:
Maybe a better name? `base-proof-directory`?